docs/platforms/native/guides/minidumps/index.mdx

```diff
@@ -111,6 +111,59 @@ curl -X POST \
 For the full list of supported values, see [_Event Payloads_](https://develop.sentry.dev/sdk/event-payloads/) and linked
 documents.
 
+## Uploading a Compressed Minidump
+
+You can compress your minidump using `gzip` if its size exceeds the [limits](#size-limits) in the next section.
+
+In this case, you would `POST` the minidump as a gzip-encoded binary:
+
+```bash
+gzip mini.dmp
+
+curl -X POST \
+  '___MINIDUMP_URL___' \
+  -H 'content-type:application/x-dmp' \
+  -H 'content-encoding: gzip' \
+  --data-binary @mini.dmp.gz
+```
+
+Since `curl` doesn't allow adding multipart form fields to "data" `POST` requests, you can't easily pass additional data as fields as in the previous section's examples.
+
+To get around this, you must construct the multipart content manually and compress it as a whole:
+
+```bash
+# Use a consistent multipart boundary throughout the example 
+# and ensure it is a unique string inside the entire multipart body
+printf -- '--boundary_minidumpXYZ\r\n' > multipart_body.txt
+
+# Add additional data via JSON or using the bracket syntax (the latter requiring a field for each entry)
+printf -- 'Content-Disposition: form-data; name="sentry"\r\n\r\n' >> multipart_body.txt
+printf -- '{"release":"my-project-name@2.3.12","tags":{"mytag":"value"}}\r\n' >> multipart_body.txt
```

Comment:
My first time going through these steps I didn't change this "release" to the actual version that the .dmp file was built for, which I think is the reason I got an `{"detail":"invalid minidump"}%` result. 

Not sure if the advanced users that would use this method of sending minidumps would make the same mistake, but still might be something worth highlighting.

Other than that, these instructions seem fine as a beginner's guide.